### DISCUSSION TITLE
Modal & Drawer

### DISCUSSION THREAD

**Discussion by KRordel** [saadeghi/daisyui#677] [Q&A] [answered]
Hi! I have some issues when I try to use modal and drawer together. When I open modal, modal`s overlay not cover drawer.

https://play.tailwindcss.com/q55BKsMmPp 

Pls, help!

**Comment by saadeghi** [comment, marked as answer]
You see a `<!-- Put this part before </body> tag -->` comment on your code?

Move that section to the end of file. example:
https://play.tailwindcss.com/V2Lkr5gVx7

**Reply by KRordel** [reply]
Oh sht! You're right! I'm ashamed that I didn't understand this. Thanks!
